File: C/Ex. 16.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
    float l1, l2, l3;

    printf ("Digite o valor de um dos lados do triangulo: ");
    scanf ("%f", &l1);

    printf ("Digite o valor do outro lado do triangulo: ");
    scanf ("%f", &l2);

    printf ("Digite o valor do ultimo lado do triangulo: ");
    scanf ("%f", &l3);

    if (l1>(l2+l3))
    {
        printf ("Nao forma um triangulo");
    }
    else
    {
        if (l2>(l1+l3))
        {
            printf("Nao forma um triangulo");
        }
        else
        {
            if (l3>(l1+l2))
            {
                printf("Nao forma um triangulo");
            }
            else
            {
                if (l1==l2 && l2==l3)
                {
                     printf ("Triangulo Equilatero");
                }
                else
                {
                    if (l1==l2 || l1==l3 || l2==l3)
                    {
                     printf ("Triangulo isosceles");
                    }
                    else
                    {
                        if (l1!=l2 && l1!=l3 && l2!=l3)
                        {
                        printf ("Triangulo Escaleno");
                        }
                    }
                }           
    
            }
        }
    }
}

File: Python/Ex 16.py
a = float(input("Digite o primeiro lado do triangulo: "))
b = float(input("Digite o segundo lado do tringulo: "))
c = float(input("Digite o terceiro lado do triangulo: "))

if a > (b + c):
    print("nao forma um tringulo")
else:
    if b > (a + c):
        print("nao forma um tringulo")
    else:
        print("nao forma um triangulo")
        if ((a == b) and (b == c)):
            print("trinagulo equilatero")
        else:
            if ((a == b) or (a == c) or (b == c)):
                print("tringulo isosceles")
            else:
                if ((a != b) or (a != c) or (b != c)):
                    print("tringulo escaleno")